styles.css

```diff
@@ -56,7 +56,16 @@ nav {
   height: 32px;
   margin-top: 0.28rem;
 }
-
+.reset {
+  margin-left: 90px;
```

Comment:
on large-width screens (more 1300px width) buttons container is starting to drift left and looks mis-aligned compared to  other content
on narrow screens it is the opposite effect - drifting to right

I assume it is because of fixed margin. Try to find another approach to align `#filter-btns` div

Look into our CSS codebase, perhaps you will find better approach when looking how other containers aligned